src/geode/basic/progress_logger.cpp

```diff
@@ -42,36 +43,42 @@ namespace geode
         Impl( std::string message, index_t max_number )
             : message_{ std::move( message ) },
               max_number_( max_number ),
-              start_{ absl::Now() }
+              start_time_{ absl::Now() },
+              current_time_{ start_time_ }
         {
+            Logger::info( message_, " starts" );
         }
 
         ~Impl()
         {
             if( current_ == max_number_ )
             {
-                Logger::info( message_, " complete" );
+                Logger::info( message_, " complete in ",
```

Comment:
What is the output of absl::FormatDuration? Is it only a number or does it return also time unit (s, min or h)?

Reply:
It depends on the duration, for duration < to 1s it displays ms or ns, for more it displays h,m,s when possible. Here is an example of our test
[2022-02-12 14:47:37.230] [console] [info] Cool message complete in 4.336392777s